packages/cms-cli/bin/cli.js

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env node
+
+const yargs = require('yargs');
+const updateNotifier = require('update-notifier');
+
+const { logger } = require('@hubspot/cms-lib/logger');
+const { logErrorInstance } = require('@hubspot/cms-lib/errorHandlers');
+
+const { setLogLevel, getCommandName } = require('../lib/commonOpts');
+const { trackHelpUsage } = require('../lib/usageTracking');
+const pkg = require('../package.json');
+
+const removeCommand = require('../commands/remove');
+
+const SCRIPT_NAME = 'banjo';
+const notifier = updateNotifier({ pkg });
+
+notifier.notify({
+  shouldNotifyInNpmScript: true,
+});
+
+const argv = yargs
+  .scriptName(SCRIPT_NAME)
+  .usage('Tools for working with the HubSpot CMS')
+  .middleware([setLogLevel])
+  .exitProcess(false)
+  .fail((msg, err /*, _yargs*/) => {
+    if (msg) logger.error(msg);
+    if (err) logErrorInstance(err);
+  })
+  .command(removeCommand)
```

Comment:
Is the thought that we'd explicitly define commands here instead of using `commandDir()`?

Reply:
Yeah, at least for now. Once we've moved off of commander, we could switch to `commandDir` if we wanted.